internal/monitors/diagnostics.go

```diff
@@ -72,19 +72,23 @@ func (mm *MonitorManager) DiagnosticText() string {
 		serviceStats := ""
 		if am.endpoint != nil {
 			serviceStats = fmt.Sprintf(
-				"Discovery Rule: %s\n"+
-					"Monitored Endpoint ID: %s\n",
+				`Discovery Rule: %s
+Monitored Endpoint ID: %s`,
 				am.config.MonitorConfigCore().DiscoveryRule,
 				am.endpoint.Core().ID)
 		}
 		activeMonText += fmt.Sprintf(
-			"%s. %s\n"+
-				"    Reporting Interval (seconds): %d\n"+
-				"%s"+
-				"    Config:\n%s\n",
+			`%s. %s
+    Reporting Interval (seconds): %d
+    Enabled Metrics:
+%s
+%s    Config:
+%s
+`,
 			am.config.MonitorConfigCore().MonitorID,
 			am.config.MonitorConfigCore().Type,
 			am.config.MonitorConfigCore().IntervalSeconds,
+			utils.IndentLines(config.ToString(am.enabledMetrics), 6),
```

Comment:
Consider making this stringify to a one-line list string instead of multiple lines, if it isn't already.

Reply:
Yes, except maybe with a `[ ... ]` around it (the YAML compact style).

Reply:
Also, this would include the "included" metrics too right?

Reply:
Yeah it includes "included" metrics as well. It's going to look messy if it wraps around though. What's the concern with laying it out vertically?

Reply:
That it is too long and requires excessive scrolling, esp for monitors with a lot of included metrics.  A lot of the work I just did on the status output was to make it more manageable and understandable.  Something like:

```
Enabled Metrics: metric1, metric2, metric3, metric4,
                             metric5, metric6
```

where it does multiple lines (not just one giant line) up to a certain column width would be nice if feasible.  We just need some formatter utility function to do that.